### PR TITLE
Adds download functionality

### PR DESCRIPTION
**Pull request by EdmundMai**
- Fixes encoding of chunks
- Adds download functionality

Still needs to test on production broker nodes

### CODE REVIEW

File: src/redux/actions/file-actions.js [hunk 1]
```diff
@@ -1,47 +0,0 @@
-const INITIALIZE_UPLOAD = "oyster/file/initialize_upload";
```
Comment (EdmundMai) [inline src/redux/actions/file-actions.js]: moved this to `upload-actions`

File: src/redux/epics/file-epic.js [hunk 2]
```diff
@@ -1,70 +0,0 @@
-import { Observable } from "rxjs";
```
Comment (EdmundMai) [inline src/redux/epics/file-epic.js]: moved to upload-epic

File: src/redux/epics/download-epic.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                ? message
+                : message.substr(0, message.length - 1);
+            const data = Iota.utils.fromTrytes(evenChars);
+            const encodedData = Encryption.decrypt(data, handle);
+            return Base64.decode(encodedData);
+          });
+
+        const arrayBuffer = _.flatten(decryptedChunks);
+        const blob = new Blob(arrayBuffer);
+        FileSaver.saveAs(blob, fileName);
```
Comment (EdmundMai) [inline src/redux/epics/download-epic.js]: will clean this up post-testnet a lol

Comment (AaronVasquez) [reply src/redux/epics/download-epic.js]: sgtm

**Pull request merged**